test/Microsoft.ML.SEAL/SEALTests.cs

```diff
@@ -0,0 +1,72 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.RunTests;
+using Microsoft.Research.SEALNet;
+using Xunit;
+
+namespace Microsoft.ML.Tests.SEAL
+{
+    public sealed class SEALTests : TestDataPipeBase
+    {
+        public SEALTests(ITestOutputHelper helper)
+            : base(helper)
+        {
+        }
+
+        private class TestClass
+        {
+            public double[] plaintext;
+            public Ciphertext[] ciphertext;
+        }
+
+        [Fact]
+        public void EncryptFeaturesTest()
+        {
+            var context = GlobalContext.CKKSContext;
+            var keygen = new KeyGenerator(context);
+
+            using (var fs = File.Open("secret.key", FileMode.OpenOrCreate))
+            {
+                keygen.SecretKey.Save(fs);
+            }
+
+            using (var fs2 = File.Open("public.key", FileMode.OpenOrCreate))
+            {
+                keygen.PublicKey.Save(fs2);
+            }
+
+            var encryptPipeline = Microsoft.ML.SEAL.SealCatalog.EncryptFeatures(true,
```

Comment:
>SealCatalog [](start = 52, length = 11)

the word catalog should not be there, please add a new catalog entry. It needs to be something like context.Transforms.Encryption.ABC(....), refer [https://github.com/dotnet/machinelearning/tree/master/docs/samples/Microsoft.ML.Samples/Dynamic/Transforms](https://github.com/dotnet/machinelearning/tree/master/docs/samples/Microsoft.ML.Samples/Dynamic/Transforms)